services/webnn/ort/context_impl_ort.cc

```diff
@@ -110,6 +110,11 @@ SessionOptions::Create(const mojom::CreateContextOptions::Device device_type) {
         /*config_value=*/"1"));
   }
 
+  CALL_ORT_FUNC(ort_api->AddSessionConfigEntry(
+      session_options.get(),
+      /*config_key=*/kOrtSessionOptionsConfigStrictShapeTypeInference,
```

Comment:
Nit: could you please add a comment explaining the usage of this option?

Reply:
Done, PTAL, thanks!

Reply:
Looks better. Could you please share the example of output shape checking you encountered?

Reply:
Done, PTAL, thanks!

Reply:
> For example, if the WebNN model
  // has a reduce op with keepdims set to 0, but the ONNX model has keepdims set
  // to 1, the session creation will fail due to inconsistent shape.

I understand this is an implementation issue. Could we explain the actual check behind that? For example, if the graph output shape set by WebNN is different from ONNX shape inference result, the session creation will fail. Apologizes for not making it super clear before.

Reply:
Done, ptal! thanks!